### PR TITLE
[dhcp_server] Adjust the dhcp_server cli data format to align with dhcp_cfggen

### PR DESCRIPTION
**Pull request by w1nda**
#### Why I did it
dhcp_server cli using string while dhcp_cfggen using array of string, I need to align them.

##### Work item tracking
- Microsoft ADO **(number only)**: 27315501

#### How I did it
Add magic character @ to make sure string will be interpreted to array of string.

#### How to verify it
Unit test.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: rules/config [hunk 1]
```diff
@@ -150,7 +150,7 @@ INCLUDE_NAT = y
 INCLUDE_DHCP_RELAY = y
 
 # INCLUDE_DHCP_SERVER - build and install dhcp-server package
-INCLUDE_DHCP_SERVER ?= n
+INCLUDE_DHCP_SERVER ?= y
```
Comment (Blueve) [inline rules/config:153]: Note: pls revert this change before merge

**Review by yaqiangz — APPROVED** [review]

**Comment by w1nda** [comment]
Hi @wangxin, cloud you help merge this PR? thanks.

**Review by yaqiangz — APPROVED** [review]

**Comment by w1nda** [comment]
Hi @yxieca a, cloud you help merge this PR? thanks.
